src/applications/claims-status/components/appeals-v2/AppealListItemV3.jsx

```diff
@@ -83,7 +83,7 @@ export default function AppealListItemV3({ appeal, name }) {
       title={appealTitle}
       subtitle={
         requestEvent &&
-        `Submitted on ${moment(requestEvent.date).format('MMMM D, YYYY')}`
+        `Received on ${moment(requestEvent.date).format('MMMM D, YYYY')}`
```

Comment:
This claim details card is used when the following is true:
- claim type is a part of the appealtypes that were passed in to YourClaimsPageV2.jsx
- useLighthouse = true
- useNewClaimCards is enabled.